superset/assets/src/visualizations/deckgl/utils.js

```diff
@@ -0,0 +1,88 @@
+import d3 from 'd3';
+import getSequentialSchemeRegistry from '../../modules/colors/SequentialSchemeRegistrySingleton';
+import { colorScalerFactory } from '../../modules/colors';
+
+export function getBreakPoints(fd, features) {
```

Comment:
I would strongly recommend listing out the fields you need for this function than the ambiguous `fd`. This will make it clearer what are needed for this function to work. 

```
export function getBreakPoints({ 
  break_points,
  metric,
  num_buckets,
}, features) {
}
```

Reply:
Cool, will do that. Thanks!